app/src/main/java/de/danoeh/antennapod/fragment/actions/FeedMultiSelectActionHandler.java

```diff
@@ -0,0 +1,189 @@
+        String[] values = new String[speeds.length + 1];
+        values[0] = SPEED_FORMAT.format(SPEED_USE_GLOBAL);
+
+        String[] entries = new String[speeds.length + 1];
+        entries[0] = activity.getString(R.string.feed_auto_download_global);
+
+        System.arraycopy(speeds, 0, values, 1, speeds.length);
+        System.arraycopy(speeds, 0, entries, 1, speeds.length);
+
+        PreferenceListDialog preferenceListDialog = new PreferenceListDialog(activity, activity.getString(R.string.playback_speed));
+        preferenceListDialog.openDialog(entries);
+        preferenceListDialog.setOnPreferenceChangedListener(pos -> {
+            saveFeedPreferences(feedPreferences -> {
+                feedPreferences.setFeedPlaybackSpeed(Float.parseFloat((String) values[pos]));
+                // TODO: 8/5/2021  
+//                EventBus.getDefault().post(
+//                        new SpeedPresetChangedEvent(feedPreferences.getFeedPlaybackSpeed(), feed.getId()));
+            });
+            
+        });
+    }
+
+    private void autoDeleteEpisodesChecked() {
+        PreferenceListDialog preferenceListDialog = new PreferenceListDialog(activity, "Auto delete episodes");
+        String[] items = new String[]{"global", "always", "never"};
+        preferenceListDialog.openDialog(items);
+        preferenceListDialog.setOnPreferenceChangedListener(which -> {
+            FeedPreferences.AutoDeleteAction autoDeleteAction = null;
+            switch (items[which]) {
+                case "global":
+                    autoDeleteAction = FeedPreferences.AutoDeleteAction.GLOBAL;
+                    break;
+                case "always":
+                    autoDeleteAction = FeedPreferences.AutoDeleteAction.YES;
+                    break;
+                case "never":
+                    autoDeleteAction = FeedPreferences.AutoDeleteAction.NO;
+                    break;
+            }
+            FeedPreferences.AutoDeleteAction finalAutoDeleteAction = autoDeleteAction;
+            saveFeedPreferences(feedPreferences -> {
+                feedPreferences.setAutoDeleteAction(finalAutoDeleteAction);
+            });
+        });
+    }
+
+    private void keepUpdatedChecked() {
+        PreferenceSwitchDialog preferenceSwitchDialog = new PreferenceSwitchDialog(activity,activity.getString(R.string.kept_updated), activity.getString(R.string.keep_updated_summary));
+        preferenceSwitchDialog.setOnPreferenceChangedListener(new PreferenceSwitchDialog.OnPreferenceChangedListener() {
+            @Override
+            public void preferenceChanged(boolean enabled) {
+                saveFeedPreferences(feedPreferences -> feedPreferences.setKeepUpdated(enabled));
+            }
+        });
+        preferenceSwitchDialog.openDialog();
+    }
+
+
+
+    private void deleteChecked() {
+        RemoveFeedDialog.show(activity, selectedItems, null);
+    }
```

Comment:
The method is only called in one place and only has one line. Please just move its contents there instead of wrapping it in a method.